_pages/04-data.md

```diff
@@ -18,50 +18,67 @@ M-Lab publishes its test data in two forms:
+* [SideStream](https://console.developers.google.com/storage/browser/archive-measurement-lab/sidestream/){:target="_blank"}
   * SideStream collects TCP state information about completed TCP connections on a system.
   * More information is available on [Github](https://github.com/npad/sidestream){:target="_blank"}.
 
 ### Inactive Tests
 
-* [Glasnost](https://console.developers.google.com/storage/browser/m-lab/glasnost/){:target="_blank"} (**deprecated**)
+* [Glasnost](https://console.developers.google.com/storage/browser/archive-measurement-lab/glasnost/){:target="_blank"} (**deprecated**)
   * Glasnost detects prioritization or censorship of network traffic.
   * More information is available at [MPI SWS](http://broadband.mpi-sws.org/transparency/bttest-mlab.php){:target="_blank"} and [Github](https://github.com/marcelscode/glasnost){:target="_blank"}.
-* [pathload2](https://console.developers.google.com/storage/browser/m-lab/pathload2/){:target="_blank"} (**deprecated**)
+* [pathload2](https://console.developers.google.com/storage/browser/archive-measurement-lab/pathload2/){:target="_blank"} (**deprecated**)
   * **M-Lab no longer supports this tool, but its archived data are available on GCS. For similar measurements with a current and supported tool, see NDT.**
   * Pathload2 measures the available bandwidth of an Internet connection.
   * More information is available at [https://code.google.com/p/pathload2-gatech/](https://code.google.com/p/pathload2-gatech/){:target="_blank"}.
-* [ShaperProbe](https://console.developers.google.com/storage/browser/m-lab/shaperprobe/){:target="_blank"} (**deprecated**)
+* [ShaperProbe](https://console.developers.google.com/storage/browser/archive-measurement-lab/shaperprobe/){:target="_blank"} (**deprecated**)
   * **M-Lab no longer supports this tool, but its archived data are available on GCS.**
   * ShaperProbe detected prioritization of network traffic.
 
+## Links to Public M-Lab Platform Data
+
+* [M-Lab Collectd](https://console.developers.google.com/storage/browser/archive-measurement-lab/utilization/){:target="_blank"}
+  * mlab-collectd is a monitoring tool for M-Lab slices, which collects resource utilization information about all M-Lab servers.
+  * More information is available on [Github](https://github.com/m-lab/collectd-mlab){:target="_blank"}.
+* [M-Lab DISCO Switch Telemetry Data](https://console.developers.google.com/storage/browser/archive-measurement-lab/switch/){:target="_blank"}
+  * Since June 2016, M-Lab has collected high resolution switch telemetry for each M-Lab server and site uplink. The **DIS**card **CO**llection (a.k.a. DISCO) dataset is also published as a standard M-Lab BigQuery table: `measurement-lab.base_tables.switch`
+  * Our [blog post announcing this dataset]({{site.baseurl}}/blog/disco-dataset/#new-disco-switch-telemetry-dataset) provides more information about the DISCO dataset.
+
+## Data for M-Lab Tests Published Elsewhere by Test Developers
```

Comment:
I think the hierarchy that might make more sense is:

Measurement Data (Active Tests)
- ETL Processed
- non-ETL Processed

Measurement Data (Platform Data)
- disco, etc

Historical / Inactive tests
- tests / links

does that make sense? I just don't want to confusingly bury BISmark and others.

Reply:
This does make sense, thanks.